app/src/main/java/com/malmstein/yahnac/views/FooterWithTopDividerLayout.java

```diff
@@ -6,11 +6,11 @@
 import android.graphics.Color;
 import android.graphics.Paint;
 import android.util.AttributeSet;
-import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
 
 import com.malmstein.yahnac.R;
 
-public class FooterWithTopDividerLayout extends LinearLayout {
+public class FooterWithTopDividerLayout extends RelativeLayout {
```

Comment:
A `RelativeLayout` is more expensive than a `LinearLayout`, there is no benefit here with the change.